init.sql

```diff
@@ -87,16 +84,10 @@ PRIMARY KEY (rideID, username, timePosted),
 FOREIGN KEY (username, timePosted) REFERENCES Bids (username, timePosted)
```

Comment:
Status should be a pre-defined value (pending, ongoing, completed), and should default as pending.